packages/fast-components-react-base/src/image/image.props.ts

```diff
@@ -35,34 +24,9 @@ export interface IImageHandledProps {
     srcSet?: string;
```

Comment:
Is `srcSet` still valid?

Reply:
it's valid on the image itself as part of the responsive images spec. It gets passed to `<img>` but not `<picture>` so I think it's valid as a handled prop.